src/content/add-to-app/ios/project-setup.md

```diff
@@ -152,108 +156,103 @@ some/path/
 
-<li>
+1. For each [Podfile target][] that needs to embed Flutter,
+   add a call to `install_all_flutter_pods(flutter_application_path)`.
 
-For each [Podfile target][] that needs to
-embed Flutter, call `install_all_flutter_pods(flutter_application_path)`.
+   ```ruby title="MyApp/Podfile"
+   target 'MyApp' do
+     install_all_flutter_pods(flutter_application_path)
+   end
+   ```
 
-```ruby title="MyApp/Podfile"
-target 'MyApp' do
-  install_all_flutter_pods(flutter_application_path)
-end
-```
+1. In the `Podfile`'s `post_install` block,
+   add a call to `flutter_post_install(installer)`.
 
-</li>
+   ```ruby title="MyApp/Podfile"
+   post_install do |installer|
+     flutter_post_install(installer) if defined?(flutter_post_install)
+   end
+   ```
 
-<li>
+   The `flutter_post_install` method adds build settings to support
+   native Apple Silicon `arm64` iOS simulators.
+   Include the `if defined?(flutter_post_install)`
+   check to validate your `Podfile` if you are running on older versions
+   of Flutter that lack this method.
 
-In the `Podfile`'s `post_install` block, call `flutter_post_install(installer)`.
+1. Use the `flutter pub get` command to refresh plugins.
 
-```ruby title="MyApp/Podfile"
-post_install do |installer|
-  flutter_post_install(installer) if defined?(flutter_post_install)
-end
-```
+   When you change the Flutter dependencies in the `pubspec.yaml` file,
+   run `flutter pub get` in your Flutter module directory.
+   This refreshes the list of plugins that the `podhelper.rb` script reads.
 
-:::note
-The `flutter_post_install` method (added in Flutter 3.1.0),
-adds build settings to support native Apple Silicon `arm64` iOS simulators.
-Include the `if defined?(flutter_post_install)` check to ensure your `Podfile`
-is valid if you are running on older versions of Flutter that don't have this method.
-:::
+1. Run the `pod install` command to embed the plugins and frameworks.
 
-</li>
+   From your iOS app project at `/path/to/MyApp`, run `pod install`.
 
-<li>
+   The `podhelper.rb` script embeds your plugins,
+   `Flutter.framework`, and `App.framework` into your project.
 
-Run `pod install`.
+   Your app's **Debug** and **Release** build configurations embed
+   the **Debug** or **Release** [build modes of Flutter][], respectively.
 
-:::note
-When you change the Flutter plugin dependencies in
-`my_flutter/pubspec.yaml`, run `flutter pub get`
-in your Flutter module directory to refresh the list
-of plugins read by the `podhelper.rb` script.
-Then, run `pod install` again from
-your application at `some/path/MyApp`.
-:::
+   To test in profile mode, add a **Profile** build configuration to your app.
 
-</li>
-</ol>
+   :::tip
+   `Flutter.framework` is the bundle for the Flutter engine,
+   and `App.framework` is the compiled Dart code for this project.
+   :::
 
-The `podhelper.rb` script embeds your plugins,
-`Flutter.framework`, and `App.framework` into your project.
+1. Open `MyApp.xcworkspace` in Xcode.
```

Comment:
A very common mistake is to open `MyApp.xcodeproj` instead of `MyApp.xcworkspace`: the `.xcodeproj` won't build as it doesn't have CocoaPod dependencies. Here's a recent user interview that specifically called this out as a top pain point: [Google internal](https://docs.google.com/document/d/1X9dNGdhBIxKXnHxGE1UqRZ0TOwmsNZw4svdu3JWZP_o/edit?usp=sharing&resourcekey=0-HTwX-TDNA80Uf7-TLjTYKw).

Your app won't have an `.xcworkspace` if it wasn't using CocoaPods before; the `.xcworkspace` is created by `pod install`. These developers will need to switch their workflow to open the `.xcworkspace` instead of the `.xcodeproj`.

Should we add more details to this section to call this out?